src/App.js

```diff
@@ -15,7 +15,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          Welcome to Cyber Shiksha Portal
```

Comment:
Please change it to online cyber Shiksha portal

Reply:
Done Changes.